presentations/2023-08-02_mlcsu-ksn-meeting/index.qmd

```diff
@@ -0,0 +1,165 @@
+---
+title: "Travels with R and Python - the power of data science in healthcare"
+subtitle: "[Chris Beeley](mailto:chris.beeley1@nhs.net)"
+format:
+  revealjs:
+    theme: [default, ./su_presentation.scss]
+    transition: none
+    chalkboard:
+      buttons: false
+    preview-links: auto
+    logo: logo_yellow.svg
+    slide-number: false
+    auto-animate: true
+---
+
+[ds_presentations]: https://the-strategy-unit.github.io/data_science/presentations/2023-08-02_mlcsu-ksn-meeting
+
+## What is data science?
+
+* "A data scientist knows more about computer science than the average statistician, and more about statistics than the average computer scientist"
+
+## Drew Conway's famous Venn diagram
+
+<img src="Data_Science_VD.png" alt="Data science Venn diagram"></a>
+
+[Source](http://drewconway.com/zia/2013/3/26/the-data-science-venn-diagram)
+
+## What are the skills of data science?
```

Comment:
Write acronyms in full. Not necessary for things like SQL or DevOps but for ML, Stats and RAP as these are used by a wider range of people. Things like RAP could be better written as RAP (Reproducible Analytical Pipelines).

Reply:
I'd rather not to get the slides snappy. I explain all the terms on the screen in the talk